stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluent-bit
+  namespace: {{ .Values.namespace }}
```

Comment:
This should be left out of the yamls and passed to `helm install --namespace foo` on the CLI.